### DISCUSSION TITLE
HDD Stuck at 100% and Chia stops farming

### DISCUSSION THREAD

**Discussion by ARxExtazy** [Chia-Network/chia-blockchain#7734] [Q&A]
Hi guys,  about a week ago i started having this problem and cant find any solution
sometimes when i go to check my Chia software i found thath the program stops synching and a random HDD is stuck at 100% on task manager.
All turn back to normal  when i turn off and on the same hdd

Any suggestion?

**Comment by madrigo64** [comment]
defragmentation or checking of errors by OS?
to disable checking you can use this command
`chkntfs /X C:`   where  C: is letter of driver
